File: matlab/kosul_ve_donguler/1_if_konu_anlatim.m
x = 5; %Belirlediðimiz Sayý
y = input('1-10 Arasý Sayý Tahmin Edin: ');
if x > y
    fprintf('Daha büyük tahminde bulunun.\n');
elseif x < y
    fprintf('Daha küçük tahminde bulunun.\n');
else
    fprintf('Doðru tahmin ettiniz.\n');
end

File: python/kosul_ve_donguler/1_if_konu_anlatim.py
#!/usr/bin/env python
__author__     = "Tunahan Bilgiç"
__copyright__  = "kodlamaogreniyorum.com, Copyright 2020"

x = 5  # Belirlediğimiz Sayı
y = int(input("1-10 Arası Sayı Tahmin Edin: "))
if x > y:
    print("Daha büyük tahminde bulunun.\n")
elif x < y:
    print("Daha küçük tahminde bulunun.\n")
else:
    print("Doğru tahmin ettiniz.\n")